DocSum/docsum.py

```diff
@@ -129,11 +182,15 @@ async def handle_request(self, request: Request, files: List[UploadFile] = File(
             file_summaries = []
             if files:
                 for file in files:
-                    file_path = f"/tmp/{file.filename}"
+                    # Fix concurrency issue with the same file name
+                    # https://github.com/opea-project/GenAIExamples/issues/1279
+                    uid = str(uuid.uuid4())
+                    file_path = f"/tmp/{uid}"
```

Comment:
If you for some reason cannot just use `file.filename` directly in `read_text_from_file()`, at least use the temporary file functionality from `aiofiles` instead of inventing your own: https://pypi.org/project/aiofiles/